src/pages/mine.tsx

```diff
@@ -29,8 +29,9 @@ const Mine: FC = () => {
     isLoading,
     callAPI
   } = useFetch(TASKS_URL, {}, false);
-  const { isLoggedIn, isLoading: isAuthenticating } = useAuthenticated();
-
+  const context = useAppContext();
+  const { state } = context || {};
```

Comment:
Wouldn't it be better to initiate the context with {} instead of declaring it everywhere?
Are there any reason for going with this approach?

Reply:
Made the change